src/acanban/templates/base.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="style.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+	<link href="https://fonts.googleapis.com/css2?family=Public+Sans&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+	<title>{{title}}</title>
+</head>
+<body>
+  {% if error %}
+  <p>error: {{ error }}</p>
+  {% endif %}
+	<header>
+   <div class="navbar">
+     <div class="navbar__logo">
+       <a href="index.html"><h3 class="logo">acanban.</h3></a>
+     </div>
+     <div class="navbar__btn">
+       <div class="navbar__links"></div>
+       <div class="navbar__sign">
+        {% if current_user.is_authenticated %}
+        <a href="" class="navbar__username">Username</a>
+        <a href="#" class="btn transparent">Log out</a>
+        {% else %}
+         <a href="#" class="btn transparent">Log in</a>
+         <a href="#" class="btn solid">Sign up</a>
+       </div>
```

Comment:
You have to end the block here.
```suggestion
       {% endif %}
       </div>
```